### DISCUSSION TITLE
html tags in i18n component

### DISCUSSION THREAD

**Discussion by judge330** [fluent-vue/fluent-vue#759] [Q&A] [answered]
Hey!
Is there a way to use simple html tags in <i18n> component like with v-html?

Example:

Vue-Template:
```
<i18n path="general-register-info" tag="span">
    <template #showTermsModalSpan="{ generalPagesTerms }">
       <span class="underline cursor-pointer" @click.prevent="showModal('terms')">{{ generalPagesTerms }}</span>
    </template>
</i18n>
```

Imported ftl file with `<br>` and `<strong>`

```
general-pages-terms = Terms

general-register-info =
    Register here.<br> But respect <strong>our terms</strong> (see {$showTermsModalSpan}).
    .general-pages-terms = { general-pages-terms }
 ```

Result:
Register here.`<br>` But respect `<strong>`our terms`</strong>` (see <ins>Terms</ins>).

Expected:
Register here.
But respect **our terms** (see <ins>Terms</ins>).

**Comment by Demivan** [comment, marked as answer]
Hi.
There is no such functionality yet. I did not have a translation that needed to mix `<i18n>` and `v-html`.
But it is a nice feature to have. Converted this into an issue: #760
